### PR TITLE
Vulnerabilities by Snyk react router dom v6 

### PR DESCRIPTION
**Pull request by kslazykv**
[#616](https://github.com/orgs/equinor/projects/721/views/28?filterQuery=&pane=issue&itemId=83609328&issue=equinor%7Ccc-toolbox%7C616)

### CODE REVIEW

File: src/modules/InvitationForPunchOut/InvitationForPunchOut.tsx [hunk 1]
```diff
@@ -32,31 +43,22 @@ const InvitationForPunchOut = (): JSX.Element => {
...
-                                    <Helmet>
-                                        <title>{'- Create'}</title>
-                                    </Helmet>
-                                    <CreateIPO />
-                                </>
-                            }
-                        />
+
+                        <Route path="/CreateIPO" element={<CreateIPOLayout />}>
+                            <Route path="" element={<CreateIPO />} />
```
Comment (larskay) [inline src/modules/InvitationForPunchOut/InvitationForPunchOut.tsx:48]: You have an empty path here, generally not usefull and can lead to unexpected behavior.

File: src/modules/InvitationForPunchOut/views/SearchIPO/SearchIpo.tsx [hunk 2]
```diff
@@ -313,9 +313,15 @@ const SearchIPO = (): JSX.Element => {
                         return response;
                     });
             } catch (error) {
-                console.error('Get IPOs failed: ', error.message, error.data);
-                // setDataLoading(false);
-                if (!error.isCancel) {
+                if (error.isCancel) {
+                    console.log('Request was cancelled');
+                } else {
+                    // setDataLoading(false);
```
Comment (larskay) [inline src/modules/InvitationForPunchOut/views/SearchIPO/SearchIpo.tsx:319]: You sure you want this commented out?

Comment (kslazykv) [reply src/modules/InvitationForPunchOut/views/SearchIPO/SearchIpo.tsx]: you are right my mistake

File: src/modules/InvitationForPunchOut/InvitationForPunchOut.tsx [hunk 3]
```diff
@@ -32,31 +36,22 @@ const InvitationForPunchOut = (): JSX.Element => {
...
-                            element={
-                                <>
-                                    <Helmet>
-                                        <title>{'- Create'}</title>
-                                    </Helmet>
-                                    <CreateIPO />
-                                </>
-                            }
-                        />
+                                path="All%20projects"
```
Comment (larskay) [inline src/modules/InvitationForPunchOut/InvitationForPunchOut.tsx:43]: You sure you want a space in the path? And is it necessary?

Comment (kslazykv) [reply src/modules/InvitationForPunchOut/InvitationForPunchOut.tsx:43]: I checked both versions, with and without spaces, and the current version is correct.

**Pull request merged**